shtdlib.sh

```diff
@@ -138,9 +141,28 @@ elif [ "${os_family}" == 'Alpine' ]; then
     os_name='alpine'
 fi
 
-# Store local IP addresses (not localhost)
+# Gets local IP addresses (excluding localhost)
+function get_local_ip_addresses {
+    local ip_addrs
+    ip_addrs=$( (
+        whichs ip && ip -4 addr show
+    ) || (
+        whichs ifconfig && ifconfig
+    ) || (
+        awk '/32 host/ { print "inet " f } {f=$2}' <<< \"$(</proc/net/fib_trie)\"
```

Comment:
Do the doulble quotes at the end need to be escaped?

Reply:
Double quotes did not need to be escaped, is an artifact from the previous implementation and is now removed.